### ISSUE TITLE
Would you mind share resnet34.caffemodel?

### ISSUE THREAD

**Issue by 408550969** [cvjena/cnn-models#14]
I have trained the resnet34,but the  accuracy only 62.2  (top-1)   my picture size is 224*224

**Comment by 408550969** [comment]
What data argument do you use when training? Would you mind share it? Thanks

**Comment by MarcelSimon** [comment]
Please see https://github.com/cvjena/cnn-models/issues/9 for a discussing of possible reasons. I simply resized all images to smaller side 256 using the convert tool. I did not use LMDB, which causes lower performance for some people.

**Comment by 408550969** [comment]
Thanks!
Will adding "BatchNorm" and "scale" layers before "conv1" affects the speed? I trained only for data migration, if  I use the Resnet for SSD, will the "BatchNorm"and“scale” aspect the results?

**Comment by MarcelSimon** [comment]
BatchNorm and Scale before conv1 is mostly for convenience and might also improve accuracy a tiny bit, but I didn't test this in detail.

**Comment by 408550969** [comment]
Thanks! Now I am training resnet-34, I use the learning rate of 0.1, I have  run 9 epoch, the accuracy of top-1 is 31% , with your experience,  is it normal?

**Comment by 408550969** [comment]
Thanks!
